src/main/java/de/relimit/commons/markdown/blockelement/BlockElementNodeBuilder.java

```diff
@@ -83,21 +85,15 @@ protected BlockElement gateKeep(BlockElement element) {
 	// Lists
 
 	public OrderedListBuilder<B> startOrderedList() {
-		final int indentationLevel = getElement().getIndentationLevel();
-		final OrderedList list = new OrderedList(indentationLevel);
-		return new OrderedListBuilder<>(list, this::append);
+		return new OrderedListBuilder<>(new OrderedList(), this::append);
 	}
 
 	public UnorderedListBuilder<B> startUnorderedList() {
-		final int indentationLevel = getElement().getIndentationLevel();
-		final UnorderedList list = new UnorderedList(indentationLevel);
-		return new UnorderedListBuilder<>(list, this::append);
+		return new UnorderedListBuilder<>(new UnorderedList(), this::append);
```

Comment:
Just for confirmation: Are these changes correct? Or are the three removed lines the new changes and the supposed changed one line was the old one? (I am thinking about the merge conflict before)